spec/models/game_spec.rb

```diff
@@ -43,4 +43,47 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      expect(game.pieces.count).to eq 32
+    end
+
+    it "should successfully hook up the database relationship between pieces and games" do
```

Comment:
Yes, I also doubted it. It should be in create-connection-between-models. will drop this test.
thanks.